overlay-sample/overlay_sample.cpp

```diff
@@ -1014,14 +1017,27 @@ bool Intersects(const XrPosef& source, const XrPosef& planePose, const XrExtent2
     return (planeExtents.width / 2.f) >= distance && (planeExtents.height / 2.f) >= distance;
 }
 
+XrPosef ApplyContentSpaceRotationalOffsetToPose(const XrPosef& Pose, const Vector3f& Axis, float Radians) {
+    Vector3f Position(Pose.position.x, Pose.position.y, Pose.position.z);
+    Quaternionf Orientation(Pose.orientation.w, Pose.orientation.x, Pose.orientation.y, Pose.orientation.z);
+    AngleAxisf Offset(Radians, Axis);
+    Vector3f OffsetPosition = Offset * Position;
+    Quaternionf OffsetOrientation = Quaternionf(Offset) * Orientation;
+    return XrPosef{
+        {OffsetOrientation.x(), OffsetOrientation.y(), OffsetOrientation.z(), OffsetOrientation.w()},
+        {OffsetPosition.x(), OffsetPosition.y(), OffsetPosition.z()}
+    };
+}
+
 //----------------------------------------------------------------------------
 // Main
 //
 void usage(const char *programName)
 {
     std::cerr << "usage: overlay-sample [options]\n";
     std::cerr << "options:\n";
-    std::cerr << "    --placement #          Set overlay layer level to # [default 0]\n";
+    std::cerr << "    --placement #          Set overlay layer level to                                                           # [default 0]\n";
```

Comment:
Heh, the "#" is supposed to represent a number.  So maybe should be "N" instead?  Then the code for these two would be:
```
std::cerr << "    --placement N          Set overlay layer level to N                                                           [default 0]\n";
std::cerr << "    --rotational_offset N  Angle in radians to offset the layer clockwise about the stage space world up vector   [default 0]\n";
```

Reply:
That is to say, the `#` that was after `--placement` moved way to the right, but it should have remained right after `--placement`.

Reply:
Tell you what, I'll just merge it and fix it, it's such a small fix!